janusgraph-cassandra/src/test/java/org/janusgraph/CassandraStorageSetup.java

```diff
@@ -144,7 +155,11 @@ public static String cleanKeyspaceName(String raw) {
 
     private static ModifiableConfiguration enableSSL(ModifiableConfiguration mc) {
         mc.set(AbstractCassandraStoreManager.SSL_ENABLED, true);
-        mc.set(STORAGE_HOSTS, new String[]{ "localhost" });
+        if (getTestStorageHost() != null) {
+            mc.set(STORAGE_HOSTS, new String[] { getTestStorageHost() });
+        } else {
+            mc.set(STORAGE_HOSTS, new String[]{ "localhost" });
```

Comment:
if getTestStorageHost is required, then we don't need to special case localhost.